doc/rados/operations/crush-map-edits.rst

```diff
@@ -271,22 +272,22 @@ and one rack bucket. The OSDs are declared as items within the host buckets::
 
 .. topic:: Bucket Types
 
-   Ceph supports four bucket types, each representing a tradeoff between
+   Ceph supports five bucket algorithms, each representing a tradeoff between
```

Comment:
i think type is better than algorithm here.

Reply:
OK.